controllers/operators/operators_objects.go

```diff
@@ -0,0 +1,206 @@
+package operators
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sv1b2 "github.com/RedHatInsights/strimzi-client-go/apis/kafka.strimzi.io/v1beta2"
+	lv1beta1 "github.com/ViaQ/loki-operator/api/v1beta1" //TODO replace ViaQ repo by grafana/loki/operator one when released
+	gv1alpha1 "github.com/grafana-operator/grafana-operator/v4/api/integreatly/v1alpha1"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	ov1 "github.com/operator-framework/api/pkg/operators/v1"
+	ov1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func operatorGroup(name string, namespace string) *ov1.OperatorGroup {
+	return &ov1.OperatorGroup{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       constants.OperatorGroup,
+			APIVersion: "operators.coreos.com/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: ov1.OperatorGroupSpec{
+			TargetNamespaces: []string{namespace},
+		},
+	}
+}
+
+func catalogSource(namespace string) *ov1alpha1.CatalogSource {
+	return &ov1alpha1.CatalogSource{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       constants.OperatorGroup,
+			APIVersion: "operators.coreos.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      constants.ObservabilityName,
+			Namespace: namespace,
+		},
+		Spec: ov1alpha1.CatalogSourceSpec{
+			DisplayName: constants.ObservabilityName,
+			Image:       "quay.io/jpinsonn/loki-operator-catalog:v0.0.1",
```

Comment:
I guess this will change but, should we anyway do it configurable?

Reply:
I would prefer to maintain our own `netobserv` catalog for dev / previews only and remove this for production

How do you see that ?

Reply:
So maybe that should be commented out until it is released?